xray.go

```diff
@@ -128,6 +129,13 @@ func WithBufferSize(bufferSize int) Option {
 	})
 }
 
+// WithBlacklist - spans with name's matching patterns in the blacklist will not be exported
```

Comment:
// WithBlacklist filters out the spans those names match one of the given regexps.